### PR TITLE
Make Tiqr app descriptions into links

### PR DESCRIPTION
**Pull request by relaxnow**
https://www.pivotaltracker.com/projects/1163646/stories/113748599

@rjkip can you review?

### CODE REVIEW

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig [hunk 1]
```diff
@@ -13,7 +13,16 @@
         </p>
         <hr>
         <p>
-            {{ ('ss.registration.selector.'~type~'.description')|trans }}
+            {{
+                ('ss.registration.selector.'~type~'.description')|trans(
+                    {
+                        '%tiqr_android_link_start%': '<a href="'~tiqrAppAndroidUrl~'">',
```
Comment (rjkip) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig]: Hmm, good to know these parameters aren't escaped :sweat_smile:.

:ship:

Comment (relaxnow) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig]: They are escaped, see ```|raw``` on: https://github.com/SURFnet/Stepup-SelfService/pull/98/files#diff-43e7ab398b2e3d7574c3dd5c0f568be9R24

There appears to be no way around ```raw``` 😢

Comment (rjkip) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig]: Good, I thought I was going mad. Well in that case, if you want to do the "correct" thing, there's also [`|replace`](http://twig.sensiolabs.org/doc/filters/replace.html) :stuck_out_tongue: (in other words: `|trans|e|replace(...)|raw`)

Comment (rjkip) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig]: But feel free to :ship:

Comment (relaxnow) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig]: What is the advantage of ```replace``` over ```trans``` arguments?

Comment (relaxnow) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/partial/secondFactor.html.twig]: Ah manual escaping with ```e```

**Pull request merged**